Source/Charts/Formatters/IValueFormatter.swift

```diff
@@ -24,12 +24,10 @@ public protocol IValueFormatter: class
     ///
     /// For performance reasons, avoid excessive calculations and memory allocations inside this method.
     ///
-    /// - returns: The formatted label ready for being drawn
+    /// - returns:                   The formatted label ready to be drawn
     ///
     /// - parameter value:           The value to be formatted
     ///
-    /// - parameter axis:            The entry the value belongs to - in e.g. BarChart, this is of class BarEntry
```

Comment:
please add correct parameter here

Reply:
I believe 'axis' was a parameter for getFormattedValue but it's not a parameter for stringForValue. This is why I removed it.

Reply:
But I will add something for the 'entry' parameter.